.github/workflows/build.yml

```diff
@@ -37,8 +37,7 @@ jobs:
         uses: actions/setup-dotnet@v3.0.3
         with:
           dotnet-version: |
-            3.1.x
-            5.0.x
+            6.0.x
```

Comment:
Should this include `7.0.x` too?

Reply:
The line below `global-json-file: "./global.json"` installs 7, so no need.